QuadradiusR/scripts/embedded_server.gd

```diff
@@ -0,0 +1,87 @@
+extends Node
+
+var SERVER_PORT_FILE = '.server_port'
+var EXECUTABLE_PATH = './quadradiusr_server'
+
+var executable_path: String
+var available: bool
+var pid: int = -1
+var wait_left: float = 0
+var server_port: int = -1
+var on_server_ready_callbacks: Array = []
+
+func _init():
+	self.available = File.new().file_exists(EXECUTABLE_PATH)
+
+	if self.available:
+		print('Server executable is available')
+	else:
+		print('Server executable is not available')
+
+
+func _ready():
+	self.run_server()
+
+
+func run_server():
+	if not self.available:
+		push_error('Server executable is not available')
+
+	var file = File.new()
+
+	if file.file_exists(SERVER_PORT_FILE):
+		Directory.new().remove(SERVER_PORT_FILE)
```

Comment:
Might be worth checking if we can use some dedicated temp data storage methods (I believe I've seen few options built-in)

Reply:
Moved to #267